pop-api/src/v0/assets/fungibles.rs

```diff
@@ -8,28 +15,22 @@ use constants::*;
 use ink::{env::chain_extension::ChainExtensionMethod, prelude::vec::Vec};
 pub use metadata::*;
 
-/// Helper method to build a dispatch call `ChainExtensionMethod` for fungibles `v0`.
-///
-/// Parameters:
-/// - 'dispatchable': The index of the module dispatchable functions.
+// Helper method to build a dispatch call `ChainExtensionMethod` for fungibles `v0`.
```

Comment:
```suggestion
// Helper method to build a dispatch call.
```

Its obvious that its building that based on the return type and that its in v0 based on the module scope. Unnecessary noise imo, and prone to error due to being overly verbose and then just copying, as is the case below.